pkg/syncer/storagepool/diskDecommissionController.go

```diff
@@ -39,9 +45,11 @@ const (
 	drainFailReasonField  = "reason"
 	drainSuccessStatus    = "done"
 	drainFailStatus       = "fail"
+	noMigrationMM         = "noMigration"
```

Comment:
Please change this to noAction because that's what we are following for other features also (like host MM).

Reply:
done.